src/Microsoft.Azure.WebJobs.Host/Scale/ConcurrencyThrottleAggregateStatus.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.WebJobs.Host.Scale
+{
+    /// <summary>
+    /// Represents an aggregated throttle result.
+    /// </summary>
+    public class ConcurrencyThrottleAggregateStatus
+    {
+        /// <summary>
+        /// Gets or sets current aggregate throttle state.
+        /// </summary>
+        public ThrottleState State { get; set; }
+
+        /// <summary>
+        /// Gets or sets the collection of currently enabled throttles. May be null.
+        /// </summary>
+        public ICollection<string> EnabledThrottles { get; set; }
```

Comment:
I've declared it nullable here, as well as in ConcurencyThrottleStatus. In adding the annotation context pragma to relevant consumer files, this had a bit of a cascade effect, leading me to have to update other APIs like `IConcurrencyThrottleProvider.GetStatus(ILogger? logger = null)` because the warning context was generating new warnings.